src/main/java/XMPLayer.mdl

```diff
@@ -45,6 +45,12 @@ type MainXMPPackage extends XMPPackage {
 
     % value of dc:title
     property dc_title: String;
+    
+    % true, if the metadata contains WTPDF Accessibility declaration
```

Comment:
We might have more WTPDF declarations in the future. They will be identified by the trailing part (after #) of the URI. Maybe we can make this more generic, eg., introducing comma-separated list  for WTPDF declarations or or even a comma-separated list of all PDF declarations (values of `conformsTo` property)?